class CommandAlreadyExists(Exception):
    pass

class ValidationError(Exception):
    pass